tests/github-retag-and-push.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+
+function finish() {
+    docker stop registry
+}
+
+trap finish EXIT
+
+docker run --detach --publish 5000:5000 --rm --name registry registry:2
```

Comment:
I am not sure why https://github.com/libero/scripts/blob/master/tests/github-push-image.sh (and by extension this) wouldn't run in CI. Should be added to https://github.com/libero/scripts/blob/master/.github/workflows/ci.yml?

Reply:
They appear to already be run by ci.yml.